src/main/java/net/earthcomputer/clientcommands/features/EnchantmentCracker.java

```diff
@@ -450,6 +454,26 @@ public boolean condition() {
                                     }
                                     return super.condition();
                                 }
+
+                                @Override
+                                public void onCompleted() {
+                                    Minecraft.getInstance().player.playNotifySound(SoundEvents.NOTE_BLOCK_PLING.value(), SoundSource.PLAYERS, 1.0f, 2.0f);
+                                }
+
+                                @Override
+                                protected void onItemThrown(int current, int total) {
+                                    MutableComponent builder = Component.empty();
+                                    int color = Mth.hsvToRgb(current / (total * 3.0f), 1.0f, 1.0f);
+                                    builder.append(Component.literal("[").withColor(ChatFormatting.GRAY.getColor()));
+                                    builder.append(Component.literal("~" + Math.round(100.0 * current / total) + "%").withColor(color));
```

Comment:
You might want to cast the `Math.round` back to an `int` just so you don't get the `.0` in the string.